modules/salt/pages/formula-saltboot.adoc

```diff
@@ -107,64 +107,27 @@ Partition Flags: boot
 
 Leave the other fields empty.
 
+[[saltboot-formula-disk-selection]]
+== Disk selection in saltboot formula
 
+When there is only one disk present on target hardware (including USB drives), an asterisk [systemitem]``*`` can be used to automatically select the disk device.
 
-== Customize the Client Boot Process
-
-You can change the client boot process using Salt pillars.
-Two Salt pillars allow you to change the protocol and server used to download the image.
-
-* The ``saltboot_download_protocol`` pillar specifies which protocol should be used to download the image to the client.
-This overrides the default protocol specified in the image pillar.
-Allowed values are ``http``, ``https``, ``ftp``, or ``tftp``.
-* The ``saltboot_download_server`` pillar specifies which server to use to download the image.
-This overrides the default hostname specified in the image pillar.
-
-
-
-.Example: Changing the Saltboot Image Download Protocol
-This example changes the protocol used for all clients.
-
-Edit the ``/srv/pillar/top.sls`` file:
+When there are multiple disks, asterisk [systemitem]``*`` can be used in device path to differentiate i.e. SATA disks from USB disks:
 
 ----
-base:
-  '*':
-    - saltboot_proto
+/dev/disk/by-path/*-ata-1
+/dev/disk/by-path/*usb*
 ----
 
-Edit the ``/srv/pillar/$branch_prefix.sls`` file:
-
-----
-saltboot_download_protocol: http
-# can be http, https, ftp, tftp
-----
-
-
+If the entered value does not contain "[systemitem]``/``", the entered value is automatically prepended by "[path]``/dev/disk/by-path/``". That means that i.e. [systemitem]``\*usb*`` is the same as [path]``/dev/disk/by-path/\*usb*``.
```

Comment:
```suggestion
If the entered value does not contain "[systemitem]``/``", the entered value is automatically prepended by "[path]``/dev/disk/by-path/``". 
For example, [systemitem]``\*usb*`` is the same as [path]``/dev/disk/by-path/\*usb*``.
```

Reply:
As an aside: "ie" means "that is", and "eg" means "for example". Moot point, though, since we don't use Latin abbreviations in our docs as a rule ;)

Reply:
:facepalm: I always thought they are 'example given' and 'in example'. Now that is embarrassing. :smile: